airbyte-config/init/build.gradle

```diff
@@ -80,4 +90,5 @@ class IconValidationTask extends DefaultTask {
 }
 
 task validateIcons(type: IconValidationTask)
+check.dependsOn downloadCombinedConnectorCatalog
```

Comment:
TODO: for local development, maybe we want to do something else than running this every time before check